src/webchat-ui/components/presentational/Header.tsx

```diff
@@ -1,82 +1,157 @@
-import React from 'react';
-import Toolbar from './Toolbar';
-import Logo from './Logo';
-import styled from '@emotion/styled';
-import IconButton from './IconButton';
-import CloseIcon from '../../assets/baseline-close-24px.svg';
-import ThumbsUpDownIcon from '../../assets/thumbs-up-down-24dp.svg';
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+import IconButton from "./IconButton";
+import CloseIcon from "../../assets/close-16px.svg";
+import GoBackIcon from "../../assets/arrow-back-16px.svg";
+import Notifications from "./Notifications";
+import classnames from "classnames";
 
-const HeaderBar = styled(Toolbar)(({ theme }) => ({
-    boxShadow: '0 5px 18px 0 rgba(0, 0, 0, 0.08), 0 5px 32px 0 rgba(0, 0, 0, 0.08), 0 8px 58px 0 rgba(0, 0, 0, 0.08)',
-    zIndex: 2,
-    minHeight: 'auto',
-    height: theme.unitSize * 7,
-    flexBasis: theme.unitSize * 7,
-    flexShrink: 0,
-    fontSize: 16,
-    fontWeight: 700
-}))
+const HeaderBar = styled.div(({ theme }) => ({
```

Comment:
Last but not least, I would suggest to implement the new Typography component here.

Reply:
Typography is coming in another open PR from Sushmita, we can add a comment there after this is merged

Reply:
It was already merged. We could anyway do a second round.